src/components/Books.module.css

```diff
@@ -0,0 +1,33 @@
+ul.booksBlock {
+  margin: 0 0 2em;
+  padding-bottom: 2em;
+  border-bottom: 1px solid #e8e8e8;
```

Comment:
- [ ] Please create a var for the #e8e8e8 color and use it wherever necessary instead of repeating it multiple times in your code so we can comply with the project requirements.